### DISCUSSION TITLE
Can denpendency-track connect to the private vulnerabilitity intelligence such as NVD ,and the private maven repositoriy such as nexus

### DISCUSSION THREAD

**Discussion by wuruyu181** [DependencyTrack/dependency-track#1953] [Q&A]
deploying with docker in Intranet，Intranet can not connect to the Internet，Can denpendency-track connect to the private vulnerabilitity intelligence such as NVD ,and the private maven repositoriy such as nexus？

**Comment by nscuro** [comment]
Yes. Private repositories can be configured in the administration panel. This is also where custom feeds of the NVD data can be configured.

![Screenshot 2022-09-13 at 17 54 52](https://user-images.githubusercontent.com/5693141/189950284-cc372b75-bbf9-4a71-880c-713aa2037909.png)

![Screenshot 2022-09-13 at 17 59 08](https://user-images.githubusercontent.com/5693141/189950309-1ed3fc10-d333-4a72-9b84-9c229e08a7ef.png)

**Reply by mciccarone** [reply]
How can the URLs be passed in during the docker container creation ?

This will allow you to change the URLs, but it required admin logging into the Frontend, which may not be installed and deployed.
